sdcm/sct_config.py

```diff
@@ -2471,6 +2471,7 @@ def verify_configuration_urls_validity(self):  # pylint: disable=too-many-branch
                                             ('oracle_user_data_format_version', ami_id_db_oracle)]:
                 if ami_list:
                     user_data_format_versions = set()
+                    self[key_to_update] = '2'
```

Comment:
why 2?
We set it in AWS here to 3: https://github.com/scylladb/scylla-cluster-tests/blob/12fd46fa03dd3cfc786779ca5957d68a31e476c7/sdcm/cluster.py#L3390

Reply:
It makes no difference if it's not SMI image